lib/rspec/templates/spec_file_template.erb

```diff
@@ -1,7 +1,8 @@
 require "spec_helper"
 
-describe "<%= @raml.title %>" do
-  include Rack::Test::Methods<% if @examples.generate! && @examples.compose.size > 0 %>
+RSpec.describe "<%= @raml.title %>", type: :request do
+<% if @examples.generate! && @examples.compose.size > 0 %>
 
```

Comment:
Let's delete this newline - I think it may be causing there to be 2 newlines between scenarios. I know it was there before you changed the file, but was meaning to change it anyway so let's do that now.